qualang_tools/config/waveform_tools.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+
+"""
+Example of parameters and how the waveforms can be defined:
+
+drag_len = 16  # length of pulse in ns
+drag_amp = 0.1  # amplitude of pulse in Volts
+drag_del_f = - 0e6  # Detuning frequency in Hz
+drag_alpha = 1  # DRAG coefficient
+drag_delta = 2 * np.pi * (- 200e6 - drag_del_f)  # in Hz
+
+Gaussian envelope:
+drag_gauss_I_wf, drag_gauss_Q_wf = drag_gaussian_pulse_waveforms(drag_amp, drag_len, drag_len / 5, drag_alpha, drag_delta, drag_del_f, substracted=False)  # pi pulse
+
+Cosine envelope:
+drag_cos_I_wf, drag_cos_Q_wf = drag_cosine_pulse_waveforms(drag_amp, drag_len, drag_alpha, drag_delta, drag_del_f)  # pi pulse
+
+"""
+
+
+def drag_gaussian_pulse_waveforms(
+    amplitude, length, sigma, alpha, delta, detune=0, substracted=True
+):
+    """
+    Function that returns DRAG pulses that compensate for leakage and also for the AC stark shift
+    that the leakage creates to the qubit frequency.
+
+    These DRAG waveforms has been implemented following the next Refs.:
+    Chen et al. PRL, 116, 020501 (2016)
+    https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.116.020501
+    and Chen's thesis
+    https://web.physics.ucsb.edu/~martinisgroup/theses/Chen2018.pdf
+
+    Envelope type: gaussian
+
+    :param float amplitude: amplitude in volts
+    :param int length: pulse length in ns
+    :param float sigma: gaussian standard deviation
+    :param float alpha: DRAG coefficient
+    :param float detune: frequency shift to correct for AC start shift
+    :param float delta: f_21 - f_10 - detune
+    :param bool substracted: if you would like to use a substracted gaussian
```

Comment:
Change to:
:param bool substracted: If true, returns a substracted Gaussian, such that the first and last points will be at 0 volts. This reduces high-frequency components due to the initial and final points offset. Default is true.